x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestsDataLoader.java

```diff
@@ -364,11 +395,16 @@ private static void loadCsvData(
                 }
             }
         }
-        if (builder.length() > 0) {
+        if (builder.isEmpty() == false) {
             sendBulkRequest(indexName, builder, client, logger);
         }
     }
 
+    private static String quoteIfNecessary(String value) {
+        boolean isQuoted = (value.startsWith("\"") && value.endsWith("\"")) || (value.startsWith("{") && value.endsWith("}"));
+        return isQuoted ? value : "\"" + value + "\"";
```

Comment:
could extract the double quote in a var and use it in all 4 places instead of inlining it.